Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "DocStringExtensions"
+uuid = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+version = "0.4.6"
+
+[deps]
+LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+
+[extras]
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["Markdown", "Test"]
```

Comment:
Should we maybe stick to `REQUIRE` for now and wait for the automated PRs (assuming that's still the plan)?

Reply:
I still have the REQUIRE in there.

Reply:
Any reason to duplicate the dependency information though? We don't need a `REQUIRE` anymore since it's only >= 0.7 with this, so we should have just one or just the other. So far we've stuck with REQUIREs for JuliaDocs packages, waiting for the ecosystem-wide PRs.

Reply:
Sure, I can remove it. Mostly used it to flesh out some `using X` where `X` wasn't actually needed, since with a Project file loading non-deps is an error.